tools/travis.ps1

```diff
@@ -181,7 +181,11 @@ else
         $packages += Start-PSPackage  @packageParams -Type AppImage
         foreach($package in $packages)
         {
-            if($env:NUGET_KEY -and $env:NUGET_URL -and [system.io.path]::GetExtension($package) -ieq '.nupkg')
+            # Publish the packages to the nuget feed if:
+            # 1 - It's a fullBuild (already checked, for not a PR)
+            # 2 - We have the info to publish (NUGET_KEY and NUGET_URL)
+            # 3 - it's a nupkg file
+            if($isFullBuild -and $env:NUGET_KEY -and $env:NUGET_URL -and [system.io.path]::GetExtension($package) -ieq '.nupkg')
```

Comment:
After merging a PR to master, if the commit message happen to have `[Feature]` tag, we will publish the nuget package as well. Is that desired? Maybe we should use `$isDailyBuild` instead.